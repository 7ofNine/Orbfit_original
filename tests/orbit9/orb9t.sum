  1  number of Trojans in this file
 189      750.0000    24250.0000
  1  1  0.9251117382655920D+00 -0.5059752731813472D-01  0.9465911043456420D+02
 -0.3911468884825705D+00  0.9434088956543580D+02 -0.9694433261966245D-01
 -0.6727224798222990D-01  0.2969849616666608D+01  0.9855105878762381D-01
  0.2026882620629764D+03 -0.5059752731813472D-01  0.9465911043456420D+02
 -0.3911468884825705D+00  0.9434088956543580D+02 -0.9694433261966245D-01
  0.1058791549051265D+02 -0.5368715107321556D+00  0.2185857302046599D+03
  1  2  0.9251117382655920D+00  0.3182208691284061D+00  0.9439596731096682D+02
 -0.1938886652393265D+00  0.9460403268903316D+02 -0.1138415425135292D-01
  0.1408378373338743D-01 -0.1648400394112537D+00  0.9855105878762381D-01
  0.2026882620629764D+03  0.3182208691284061D+00  0.9439596731096682D+02
 -0.1938886652393265D+00  0.9460403268903316D+02 -0.1138415425135292D-01
  0.9595076683730464D-01 -0.2248287298798626D+00  0.2185857302046599D+03
  1  3  0.9251117382655920D+00  0.5504326835509981D+00  0.9433195595695939D+02
  0.3811900281544316D+00  0.9466804404304064D+02 -0.1201643203565695D+01
  0.7452837208754240D-02  0.1166464691506519D-01  0.9855105878762381D-01
  0.2026882620629764D+03  0.5504326835509981D+00  0.9433195595695939D+02
  0.3811900281544316D+00  0.9466804404304064D+02 -0.1201643203565695D+01
  0.6014534598543506D+00  0.4081096141875225D+00  0.2185857302046599D+03
 200    24375.0000    49250.0000
  1  1  0.1014762874345379D+01  0.1685295372806871D+01  0.1003475571205669D+03
  0.8813335855121340D+00  0.9965244287943310D+02  0.4357830321923690D+00
 -0.6036961220343708D-01  0.3130895710750362D+01  0.1036265370917569D+00
  0.2145104570521696D+03  0.1685295372806871D+01  0.1003475571205669D+03
  0.8813335855121340D+00  0.9965244287943310D+02  0.4357830321923690D+00
  0.1311848879424255D+02  0.8671639612620070D+00  0.2313582368580461D+03
  1  2  0.1014762874345379D+01  0.6951142411338327D+00  0.1000754749988687D+03
  0.8715660643847372D+00  0.9992452500113133D+02 -0.3206235066023419D+00
 -0.7170698296429289D-02 -0.1277873503873805D+00  0.1036265370917569D+00
  0.2145104570521696D+03  0.6951142411338327D+00  0.1000754749988687D+03
  0.8715660643847372D+00  0.9992452500113133D+02 -0.3206235066023419D+00
  0.5475737983800887D+00  0.9809876191343051D+00  0.2313582368580461D+03
  1  3  0.1014762874345379D+01 -0.8932472693989202D-01  0.8858402366744349D+02
  0.1663059221438599D+00  0.1114159763325565D+03  0.1073441909854919D+02
 -0.2382415001841601D-01  0.1167409806454618D-01  0.1036265370917569D+00
  0.2145104570521696D+03 -0.8932472693989202D-01  0.8858402366744349D+02
  0.1663059221438599D+00  0.1114159763325565D+03  0.1073441909854919D+02
 -0.5062833190789107D-01  0.1901850925516030D+00  0.2313582368580461D+03
 200    49375.0000    74250.0000
  1  1  0.9472100266857169D+00 -0.1566587807499475D+01  0.9989690246169769D+02
 -0.1265715505602454D+01  0.1001030975383023D+03 -0.3504085053232128D+00
 -0.8798088296208853D-01  0.3141258178420058D+01  0.1038759168824486D+00
  0.2142237432474159D+03 -0.1566587807499475D+01  0.9989690246169769D+02
 -0.1265715505602454D+01  0.1001030975383023D+03 -0.3504085053232128D+00
  0.9550578007101469D+01 -0.1515263470713189D+01  0.2307422414721140D+03
  1  2  0.9472100266857169D+00 -0.2061950766045121D+00  0.9956332199463658D+02
 -0.7008170106464251D+00  0.1004366780053634D+03  0.5419424461400193D+00
  0.2672974483750140D-01 -0.1879291709180650D+00  0.1038759168824486D+00
  0.2142237432474159D+03 -0.2061950766045121D+00  0.9956332199463658D+02
 -0.7008170106464251D+00  0.1004366780053634D+03  0.5419424461400193D+00
 -0.6098459179986945D+00 -0.8164343660234343D+00  0.2307422414721140D+03
  1  3  0.9472100266857169D+00 -0.2599259718524984D+00  0.9807413453509115D+02
  0.4144542710622074D+00  0.1019258654649088D+03 -0.1070527577535635D+01
  0.2374240584600152D-01  0.1658594585135878D-01  0.1038759168824486D+00
  0.2142237432474159D+03 -0.2599259718524984D+00  0.9807413453509115D+02
  0.4144542710622074D+00  0.1019258654649088D+03 -0.1070527577535635D+01
 -0.3321774986090654D+00  0.4616808957295598D+00  0.2307422414721140D+03
 200    74375.0000    99250.0000
  1  1  0.1029512345235691D+01  0.2989709973698380D+00  0.9950464391463520D+02
  0.1235295803780604D+01  0.1004953560853648D+03  0.3261410950245369D+00
 -0.5556237118757481D-01  0.3172868066785462D+01  0.1054641750646784D+00
  0.2143529135964814D+03  0.2989709973698380D+00  0.9950464391463520D+02
  0.1235295803780604D+01  0.1004953560853648D+03  0.3261410950245369D+00
  0.1151312619043789D+02  0.1262816731388761D+01  0.2310071521959888D+03
  1  2  0.1029512345235691D+01 -0.9907121707294254D+00  0.1001778125062149D+03
  0.6522821900490732D+00  0.9982218749378511D+02 -0.3634021147050176D+00
 -0.3075193318357044D-01 -0.1435404165306049D+00  0.1054641750646784D+00
  0.2143529135964814D+03 -0.9907121707294254D+00  0.1001778125062149D+03
  0.6522821900490732D+00  0.9982218749378511D+02 -0.3634021147050176D+00
 -0.1374298386567408D+01  0.7257131483866444D+00  0.2310071521959888D+03
  1  3  0.1029512345235691D+01 -0.6257874793004983D+00  0.1031684606658491D+03
 -0.5324517907606904D+00  0.9683153933415093D+02 -0.3113984832530561D+01
 -0.2404377663516354D-01 -0.2147856690003960D-01  0.1054641750646784D+00
  0.2143529135964814D+03 -0.6257874793004983D+00  0.1031684606658491D+03
 -0.5324517907606904D+00  0.9683153933415093D+02 -0.3113984832530561D+01
 -0.7085178825987403D+00 -0.5767874762227381D+00  0.2310071521959888D+03
