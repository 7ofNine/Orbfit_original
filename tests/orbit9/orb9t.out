; orbit9t vers. a, output file
  nsamp=          20  njump=          40  nsamp2=         100  njump2=           8
 in input            4  planets: KEP HEL ECLM00
barycenter corr.flag  1 no. inner pla.   4
; planetary inverse masses as used
  9.9999402303776952D-01  1.0473486309727623D+03  3.4979017677866345D+03
  2.2902950523706935D+04  1.9412259775973071D+04
  planetary initial conditions as read
   1  asteroids required,   1  with LCE; numbers:
  588      
  asteroid 588        OK
  file end looking for 588                 record            2
  Selection of the best stepsize, maximum=  0.10000000000000001     
 orbit no.  1 sugg.step= 0.1000 a,e= 5.202  0.0489
 orbit no.  2 sugg.step= 0.1000 a,e= 9.558  0.0532
 orbit no.  3 sugg.step= 0.1000 a,e=19.142  0.0497
 orbit no.  4 sugg.step= 0.1000 a,e=29.975  0.0075
 orbit no.  5 sugg.step= 0.0592 a,e= 5.206  0.2508
 min. step required  0.059159 selected  0.056818 nn=    22
  input complete, starting from t=   0.0000000000000000     
  filters parameters
           7   165.00000000000000        1.2500000000000000               49   687.50000000000000        1.2500000000000000     
 end orbit9t, time =        100000.0000
